packages/netlify-cms-core/src/backend.ts

```diff
@@ -98,67 +90,6 @@ const sortByScore = (a: fuzzy.FilterResult<EntryValue>, b: fuzzy.FilterResult<En
   return 0;
 };
 
-function createPreviewUrl(
```

Comment:
Moved to https://github.com/netlify/netlify-cms/pull/3116/files#diff-6886f49f9a11053c0ddaaa2256e60b65R141